utils/bundles.stanza

```diff
@@ -153,17 +153,19 @@ public pcb-enum ocdb/utils/bundles/SPIPins :
   SPI-CIPO
   SPI-COPI
   SPI-SDIO
+  SPI-RDY
```

Comment:
Seems very specific to the STM peripheral. I think it is fine to include it, but maybe we use a separate function to create SPI bundles with this pin include.